src/Microsoft.Data.Sqlite/Utilities/NativeLibraryLoader.cs

```diff
@@ -1,149 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-#if NET451
-
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
-using System.Runtime.InteropServices;
-
-namespace Microsoft.Data.Sqlite.Utilities
-{
-    internal static class NativeLibraryLoader
-    {
-        private const uint LOAD_WITH_ALTERED_SEARCH_PATH = 8;
-        private const short PROCESSOR_ARCHITECTURE_INTEL = 0;
-        private const short PROCESSOR_ARCHITECTURE_ARM = 5;
-        private const short PROCESSOR_ARCHITECTURE_AMD64 = 9;
-
-        [DllImport("api-ms-win-core-libraryloader-l1-1-0", SetLastError = true)]
-        private static extern IntPtr LoadLibraryEx(string lpFileName, IntPtr hFile, uint dwFlags);
-
-        [DllImport("api-ms-win-core-libraryloader-l1-1-0", SetLastError = true)]
-        private static extern IntPtr GetModuleHandle(string lpModuleName);
-
-        [DllImport("api-ms-win-core-libraryloader-l1-1-0")]
-        private static extern void GetSystemInfo(out SYSTEM_INFO lpSystemInfo);
-
-        public static bool TryLoad(string dllName)
-        {
-            if (dllName == null)
-            {
-                throw new ArgumentNullException(nameof(dllName));
-            }
-            if (IsLoaded(dllName))
-            {
-                return true;
-            }
-
-            string dllPath;
-            if (!TryFind(dllName, out dllPath))
-            {
-                return false;
-            }
-
-            var handle = LoadLibraryEx(dllPath, IntPtr.Zero, LOAD_WITH_ALTERED_SEARCH_PATH);
-            Marshal.ThrowExceptionForHR(Marshal.GetLastWin32Error());
-
-            return handle != IntPtr.Zero;
-        }
-
-        public static bool TryFind(string dllName, out string dllPath)
-        {
-            if (!dllName.EndsWith(".dll"))
-            {
-                dllName += ".dll";
-            }
-
-            dllPath = GetSearchPaths()
-                .Select(d => Path.Combine(d, dllName))
-                .FirstOrDefault(File.Exists);
-
-            return dllPath != null;
-        }
-
-        private static IEnumerable<string> GetSearchPaths()
-        {
-            yield return AppDomain.CurrentDomain.BaseDirectory;
-            yield return Path.Combine(AppDomain.CurrentDomain.BaseDirectory, GetArchitecture());
-
-            var privateBinPath = AppDomain.CurrentDomain.SetupInformation.PrivateBinPath;
-            if (!string.IsNullOrEmpty(privateBinPath))
-            {
-                foreach (var path in privateBinPath.Split(new [] { Path.PathSeparator }, StringSplitOptions.RemoveEmptyEntries))
-                {
-                    yield return path;
-                    yield return Path.Combine(path, GetArchitecture());
```

Comment:
I'm trying to navigate the layers of SQLitePCL packages. I'm sure this handled, but can you point me to which package actually has the native windows x64 vs x86 binaries? What I'm trying to make sure we've addressed this this: https://github.com/dotnet/sdk/pull/847. Microsoft.NET.Sdk will not require .NET Framework apps to specify a runtime identifier. But if the package is using the 'runtimes' folder of NuGet to distribute win-x86 vs win-x64 binaries, we may run into issues.